src/pdn/test/tempSensor.api.defok

```diff
@@ -996,6 +996,9 @@ SPECIALNETS 3 ;
       + ROUTED met4 0 + SHAPE STRIPE ( 64300 64340 ) via4_5000x5000
       NEW met4 0 + SHAPE STRIPE ( 47150 64340 ) via4_1600x5000
       NEW met4 0 + SHAPE STRIPE ( 29080 64340 ) via4_5000x5000
+      NEW met4 0 + SHAPE STRIPE ( 64300 46000 ) via4_5000x1600
+      NEW met4 0 + SHAPE STRIPE ( 47150 46000 ) via4_1600x1600
+      NEW met4 0 + SHAPE STRIPE ( 29080 46000 ) via4_5000x1600
```

Comment:
Why are there new stripes?

Reply:
I had to adjust the testcase, the behaviour before was a bit strange and didnt translate well to a more general handling.